webhook/resources/backupbackingimage/validator.go

```diff
@@ -44,13 +44,21 @@ func (bbi *backupBackingImageValidator) Create(request *admission.Request, newOb
 	}
 	backingImageName := backupBackingImage.Spec.BackingImage
 
-	backingImage, err := bbi.ds.GetBackingImageRO(backingImageName)
-	if err != nil {
-		return werror.NewInvalidError(fmt.Sprintf("failed to get the backing image %v for backup: %v", backingImageName, err), "")
+	// The backup backing image might be created by the backup target controller synchronized from the remote backup target,
+	// and the backing image might not exist in the local cluster
+	exists := false
+	if backupBackingImage.Labels != nil {
+		_, exists = backupBackingImage.Labels[types.GetLonghornLabelKey(types.BackupTargetSync)]
 	}
-	// TODO: support backup for v2 data engine in the future
-	if types.IsDataEngineV2(backingImage.Spec.DataEngine) {
-		return werror.NewInvalidError(fmt.Sprintf("backing image %v uses v2 data engine which doesn't support backup operations", backingImageName), "")
+	if !exists {
+		backingImage, err := bbi.ds.GetBackingImageRO(backingImageName)
+		if err != nil {
+			return werror.NewInvalidError(fmt.Sprintf("failed to get the backing image %v for backup: %v", backingImageName, err), "")
+		}
+		// TODO: support backup for v2 data engine in the future
+		if types.IsDataEngineV2(backingImage.Spec.DataEngine) {
+			return werror.NewInvalidError(fmt.Sprintf("backing image %v uses v2 data engine which doesn't support backup operations", backingImageName), "")
+		}
```

Comment:
I think we don't need to validate whether a backing image exists. We can allow users to create v2 backing image but always failed the backup (backupBacking.status.State=error, backupBackingImage.status.Error = "backing up v2 backing image is not supported). Then, we don't need the special label. WDYT?

cc @ChanYiLin

Reply:
Removed the label, and ignored the error `ErrorIsNotFound`.

Reply:
Sounds good to me

Reply:
After a second thought, I think using webhook is still more simple.
The current change in this PR is also good enough
If the backing image exists in the cluster and it is v2 backing image
then the webhook blocks the backup creation.

If the backing image doesn't exist in the cluster, then the backupbackingimage is pulled from the remote, we allow it.

Reply:
There are two different behaviors for v2 backup backingimage. Is making them consistent better?